pubsub/kafka/metadata.yaml

```diff
@@ -0,0 +1,128 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: kafka
+version: v1
+status: stable
+title: "Apache Kafka"
```

Comment:
Should this be Kafka?

Reply:
I think Apache Kafka is fine: https://kafka.apache.org/